pages/ja/lb4/Concepts.md

```diff
@@ -0,0 +1,33 @@
+---
+lang: ja
+title: 'キーコンセプト'
```

Comment:
according to https://raw.githubusercontent.com/strongloop/loopback.io/gh-pages/pages/ja/lb4/Concepts.md, we need:
`layout: translation` in the front matter.  Similarly for the other pages.